packages/analytics-js-plugins/src/iubendaConsentManager/index.ts

```diff
@@ -108,7 +101,7 @@ const IubendaConsentManager = (): ExtensionPlugin => ({
         if (iubendaConsentPurposes) {
           const configuredConsents = iubendaConsentPurposes.map(p => p.purpose.trim()).filter(n => n);
 
-          // Check if any of the destination's mapped ketch purposes are consented by the user in the browser.
+          // Check if any of the destination's mapped iubenda purposes are consented by the user in the browser.
           return configuredConsents.some(matchPredicate) || configuredConsents.length === 0;
         }
```

Comment:
This block is not required at all. It is only needed for the legacy CM providers.